docs/manual/catalogi/index.rst

```diff
@@ -1,6 +1,166 @@
 .. _manual_catalogi_index:
 
 ==================
-Catalog management
+Catalogusbeheer
 ==================
 
+.. note:: Om catalogussen te kunnen beheren moet je tot de **Catalogi admin**
+   groep behoren of equivalente permissies hebben. Zie
+   :ref:`manual_users_groups` voor groepenbeheer.
+
+Catalogus aanmaken
+==================
+
+Om aan de gang te kunnen met catalogusbeheer moet er eerst een catalogus aangemaakt worden,
+dit kan door via de beginpagina van de admin te klikken op **Catalogi** onder het kopje **Gegevens**
+en vervolgens op de knop **Catalogus toevoegen** rechtsbovenin te klikken.
+
+.. image:: assets/create_catalogus.png
+    :width: 100%
+    :alt: Catalogus toevoegen
+
+Op de volgende pagina dienen minimaal alle dikgedrukte velden ingevuld te worden,
+waarna er op de **OPSLAAN** knop rechtsonderin de pagina geklikt kan worden.
+
+Zaaktype aanmaken
+=================
+
+Na het toevoegen van de catalogus kan er een Zaaktype toegevoegd worden aan deze
+catalogus door te klikken op **Toon Zaaktypen** onder het kopje **ACTIES**.
+
+.. image:: assets/toon_zaaktypen.png
+    :width: 100%
+    :alt: Toon alle zaaktypen
+
+Vervolgens dient er geklikt te worden op de **Zaaktype toevoegen** knop rechtsbovenin.
+Op de volgende pagina is het aan te maken zaaktype al gekoppeld aan de juiste catalogus
+en moet alleen de overige verplichte informatie nog ingevuld worden. Als alle verplichte
+velden gevuld zijn, kan het zaaktype opgeslagen worden door te klikken op de knop
+**Opslaan en opnieuw bewerken**, onderaan de pagina
+
+.. image:: assets/zaaktype_opslaan.png
+    :width: 100%
+    :alt: Zaaktype publiceren
+
+Zaaktype publiceren
+===================
+
+Het zojuist aangemaakt zaaktype is nog een concept, wat inhoudt dat dit zaaktype
+niet gebruikt kan worden buiten de Catalogi API zelf (er kunnen bijvoorbeeld nog
+geen Zaken aangemaakt worden met dit zaaktype). Om ervoor te zorgen dat dit wel mogelijk is,
+moet het zaaktype eerst gepubliceerd worden. Dit kan door te klikken op de **Publiceren**
+knop, onderaan de detail pagina van het zaaktype.
+
+.. image:: assets/zaaktype_publiceren.png
+    :width: 100%
+    :alt: Zaaktype opslaan en opnieuw bewerken
+
+**LET OP**: als er op de detailpagina van het zaaktype aanpassingen gemaakt worden en
+er vervolgens op **Publiceren** gedrukt wordt, dan worden deze aanpassingen opgeslagen.
+
+Een nieuwe versie van een zaaktype aanmaken
```

Comment:
@sergei-maertens @joeribekker bij het aanmaken van een nieuwe versie van een zaaktype zag ik dat de `datum_begin_geldigheid` van de nieuwe versie gelijk is aan de `datum_begin_geldigheid` van het origineel. Zou `datum_begin_geldigheid` van de nieuwe versie niet gelijk moeten zijn aan `datum_einde_geldigheid` van de versie ervoor?

Reply:
Klopt, terechte opmerking. Ik heb #391 aangemaakt hiervoor